openfisca_uk/calibration/losses/loss_calculator.py

```diff
@@ -23,28 +17,17 @@ def __init__(self, sim: Microsimulation, validation_split: float = 0.1):
 
         Args:
             sim (Microsimulation): A microsimulation from which to draw demographic data.
-            validation_split (float, optional): Percentage of metrics to use as validation. Defaults to 0.1.
+            validation_split (float, optional): Percentage of non-population metrics to use as validation. Defaults to 0.1.
         """
-        self.losses = [
-            # Populations,
-            RegionalPopulations,
-            Households,
-            TenureType,
-            CouncilTaxBandHouseholds,
-            CountryLevelAggregates,
-            UKProgramCaseloads,
-            UKProgramAggregates,
-            # IncomeTaxPayersByBand,
-            # IncomeTaxRevenueByIncome,
-        ]
+        self.losses = [Households, Populations, Families, BudgetaryImpact]
         self.validation_split = validation_split
         self.sim = sim
         self.training_log = []
         self.metrics = sum(
             [list(loss.get_metric_names()) for loss in self.losses], []
         )
         non_population_metrics = sum(
-            [list(loss.get_metric_names()) for loss in self.losses[5:]], []
+            [list(loss.get_metric_names()) for loss in self.losses[2:]], []
```

Comment:
What does the `2` correspond to? Could you make it a named constant for documentation?

Reply:
Yep, and changed to a more verbose indexing method by using the actual category names.